website/docs/release_notes.md

```diff
@@ -310,7 +476,7 @@ a warning message when such non-compliant options are passed.
   in [#2207](https://github.com/VirtusLab/scala-cli/pull/2207)
 * Update guava to 32.0.1-jre by [@scala-steward](https://github.com/scala-steward-org/scala-steward)
   in [#2197](https://github.com/VirtusLab/scala-cli/pull/2197)
-* Update scala-cli.sh launcher for 1.0.1 by @github-actions in [#2194](https://github.com/VirtusLab/scala-cli/pull/2194)
+* Update scala-cli.sh launcher for 1.0.1 by [@github-actions](https://github.com/github-actions) in [#2194](https://github.com/VirtusLab/scala-cli/pull/2194)
```

Comment:
The link is incorrect, I think that's why there was none.

Reply:
y're right, my bad.
fixed it to https://github.com/features/actions